src/content/docs/fr/2/guide/upgrade-migrate/from-tauri-1.mdx

```diff
@@ -0,0 +1,1148 @@
+---
+title: Mise à Niveau Depuis Tauri 1.0
+
+i18nReady: true
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import CommandTabs from '@components/CommandTabs.astro';
+
+Ce guide vous explique comment mettre à niveau votre application basée sur Tauri 1.0 vers Tauri 2.0.
+
+## Migration Automatisée
+
+
+Tauri v2 contient une commande `migrate` qui automatise la plupart des processus et vous aide à finir votre migration:

```

Comment:
I think I would keep it simple here

```suggestion
Tauri v2 contient la commande `migrate` qui simplifie votre migration:
```

Reply:
Why not, I just followed the wording of the english documentation